Add 5.14.1 release notes (#1849)
sensu/sensu-docs: content/sensu-go/5.14/release-notes.md

---

## 5.14.1 release notes

**October 16, 2019** &mdash; The latest release of Sensu Go, version 5.14.1, is now available for download.
This release adds Prometheus gauges for check schedulers and fixes several bugs, including a bug discovered in 5.14.0 that prevented OIDC authentication providers from properly loading on start-up.

See the [upgrade guide][1] to upgrade Sensu to version 5.14.1.

**NEW FEATURES:**

- Added Prometheus gauges for check schedulers.

**FIXES:**

- ([Licensed tier][79]) Sensuctl will not incorrectly warn of entity limits for unlimited licenses.
- ([Licensed tier][79]) `oidc` authentication providers will now properly load on start-up.
- When opening a Bolt database that is already open, `sensu-agent` will not hang indefinitely.

> - Running [`sensuctl dump`][84] for multiple types as YAML will print to the specified file instead of to `STDOUT`.

Reviewer: ```suggestion
- Addresses an issue where running [`sensuctl dump`][84] for multiple resource types with the output format as YAML separators were printed to `STDOUT` instead of the specified file.
```

Reviewer: This is perhaps a bit more clear

> - Addresses an issue where running [`sensuctl dump`][84] for multiple resource types with the output format as YAML separators were printed to `STDOUT` instead of the specified file.

Reviewer: ```suggestion
- Addresses an issue where running [`sensuctl dump`][84] for multiple resource types with the output format as YAML resulted in separators were printed to `STDOUT` instead of the specified file.
```

> - Sensu will not crash with a panic due to sending on a closed channel.

Reviewer: ```suggestion
- Fixed a crash in sensu-backend (panic: send on closed channel)
```

> - Sensu will not crash with a panic due to sending on a closed channel.

Author: ```suggestion
 - Fixed a crash in sensu-backend (panic: send on closed channel)
```
